12/umbraco-forms/editor/defining-and-attaching-prevaluesources/README.md

```diff
@@ -53,6 +53,20 @@ This can be useful if the recorded entries are used in any subsequent workflows
 
 ![Prevalues with captions](images/Prevalues-with-caption.png)
 
+### Defining Cache Options for the Prevalue Source
+
+Sometimes retrieving the list of options for a prevalue source can be an expensive operation. If the source depends on data from external systems, it could be that the list changes regularly or rarely.
+
+Given the variation here, we allow you to select an appropriate level of caching for the list of options.
+
+You can choose between:
+
+- `No Caching` - no caching will be applied and the list of options will be retrieved from source on every request. You will likely only want to choose this option if the information changes frequently and it's important that the latest is presented to website visitors.
+- `Cache For Specified Time` - the list will be cached for the period of time provided.
+- `Cache With No Expiry` - the list will be cached on first request and not retrieved again until either the prevalue source is edited or the website is restarted. This ismost appropriate to use for information held within the prevalue source data itself (such as  when uploading a text file).
+
+![Prevalue cache options](../../../../10/umbraco-forms/editor/defining-and-attaching-prevaluesources/images/prevalue-cache-options.png)
```

Comment:
I'd recommend adding the image to the 12 directory as well.
If you've seen other places in the docs, where images are references in other version directories than the one the article is places in, you've seen one of the bugs we're currently experiencing with GitBook.
We do not know the extend of the bug or how it can and well affect the images - but as a rule of thumb, when adding new images, please add them to all the directories that needs to reference the images 🙌

Reply:
OK, noted - and have added.